util/SitRepEntry.h

```diff
@@ -23,12 +24,14 @@ class FO_COMMON_API SitRepEntry : public VarText {
     const std::string&  GetDataString(const std::string& tag) const;
     int                 GetTurn() const { return m_turn; }
     const std::string&  GetIcon() const { return m_icon; }
+    const std::string&  GetTemplateString() const   { return m_stringtable_lookup_flag ? m_template_string : m_label; }
```

Comment:
What is m_label for? If you are already setting m_stringtable_lookup_flag to false for non-stringtable lookup template strings, can't you always just return m_template_string ?